xkeysnail/__init__.py

```diff
@@ -1,5 +1,22 @@
 # -*- coding: utf-8 -*-
 
+def killxkeysnail(process):
+    from os import getpid
+    import re
+    import psutil
+
+    killed = None
+    try:
+        for proc in psutil.process_iter():
```

Comment:
We need to figure our what our "best practices" runtime story is... if someone is supposed to just run us from systemd then I'm not sure any of this process management stuff is necessary - systemd is more than capable of starting/stopping/signaling processes all on it's own - since we don't fork.

If someone is running us (as themselves) from say `.xinitrc` (I think that could be another way to go) then this is one line of code to do this in bash, I'm not sure we need to build the checking into the keymapper itself, etc.  I'm trying to imagine when one would have lots of extra processes running around that they had to deal with.   Sometimes I wish your commit messages were more than one liners - and explained the thinking and rational for each change. :-)

What was your use case for adding this?

Reply:
I did this exactly because using systemd it couldn't kill the process when necessary.
Another case is that the user needs it for the process that was running in the background, for whatever reason.

If there is another solution for systemd to handle this better, this can be removed.

Reply:
I opened the code yesterday in this snippet because I didn't remember for sure what it was, this snippet is from the -k command line, it has nothing to do with systemd.

The system signals are already registered elsewhere in the program.